### PR TITLE
Add minimal image

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by marquiz** [comment]
Cool 😎 Have you tested/verified the minimal image? 😉 I haven't. At least build works, though.

One thing that I'd like to see is update the documentation. At least mention the minimal image with perhaps instructions how to deploy it.

**Comment by ArangoGutierrez** [comment]
> Cool sunglasses Have you tested/verified the minimal image? wink I haven't. At least build works, though.
> 
> One thing that I'd like to see is update the documentation. At least mention the minimal image with perhaps instructions how to deploy it.

Ready for a next round @marquiz

**Comment by ArangoGutierrez** [comment]
Rebased and ready for a next round @marquiz

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ARG BASE_IMAGE_FULL
+ARG BASE_IMAGE_MINIMAL
```
Comment (marquiz) [inline Dockerfile:2]: Sorry, one nit 😬  could you move this to the first patch where it really belongs to?

Comment (ArangoGutierrez) [reply Dockerfile:2]: yup, done

**Review by marquiz — APPROVED** [review]
Looks good, now 👍 
/lgtm

**Pull request merged**